android/src/main/java/com/swmansion/rnscreens/ScreenStackFragment.kt

```diff
@@ -123,6 +130,11 @@ class ScreenStackFragment : ScreenFragment, ScreenStackFragmentWrapper {
         return view
     }
 
+    override fun onStop() {
+        mLastFocusedChild = findLastFocusedChild()
```

Comment:
```suggestion
        if (isTelevision()) mLastFocusedChild = findLastFocusedChild()
```

Reply:
Sure, thanks for the suggestion! Changed in https://github.com/software-mansion/react-native-screens/pull/1894/commits/4466f8d6df6cac984534a13781bbc6f94f0491ec.